### ISSUE TITLE
Abort error upgrading from v1.5.1 to v1.5.2

### ISSUE THREAD

**Issue by davidpesce** [LearningLocker/learninglocker#673]
**Version**
v1.5.1 to v1.5.2

**Steps to reproduce the bug**
1. Follow directions here: https://github.com/LearningLocker/learninglocker/releases/tag/v1.5.2
2. Aborts with error (below) on step three ( git merge v1.5.2)

**Expected behaviour**
No error

**Actual behaviour**
Error below

**Server information**
CentOS 7
MongoDB

**Client information**
OS: WIndows 8.1
Browser: Firefox (latest)

**Additional information**
I know I can get around this, but wanted to include this for future upgrades.
```
git merge v1.5.2
error: The following untracked working tree files would be overwritten by merge:
        public/assets/js/bower_components/FileSaver.js/FileSaver.js
        public/assets/js/bower_components/FileSaver.js/FileSaver.min.js
        public/assets/js/bower_components/FileSaver.js/demo/demo.css
        public/assets/js/bower_components/FileSaver.js/demo/demo.js
        public/assets/js/bower_components/FileSaver.js/demo/demo.min.js
        public/assets/js/bower_components/backbone.babysitter/lib/backbone.babysitter.js
        public/assets/js/bower_components/backbone.babysitter/lib/backbone.babysitter.min.js
        public/assets/js/bower_components/backbone.babysitter/src/build/backbone.babysitter.js
        public/assets/js/bower_components/backbone.babysitter/src/childviewcontainer.js
        public/assets/js/bower_components/backbone.basicauth/backbone.basicauth.js
        public/assets/js/bower_components/backbone.marionette/lib/backbone.marionette.js
        public/assets/js/bower_components/backbone.marionette/lib/backbone.marionette.min.js
        public/assets/js/bower_components/backbone.marionette/lib/core/backbone.marionette.js
        public/assets/js/bower_components/backbone.marionette/lib/core/backbone.marionette.min.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.application.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.approuter.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.behavior.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.behaviors.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.bindEntityEvents.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.callbacks.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.collectionview.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.compositeview.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.controller.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.domRefresh.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.error.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.helpers.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.itemview.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.layoutview.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.module.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.object.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.region.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.regionManager.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.renderer.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.templatecache.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.triggermethod.js
        public/assets/js/bower_components/backbone.marionette/src/marionette.view.js
        public/assets/js/bower_components/backbone.marionette/tasks/jsDocFile.js
        public/assets/js/bower_components/backbone.wreqr/lib/backbone.wreqr.js
        public/assets/js/bower_components/backbone.wreqr/lib/backbone.wreqr.min.js
        public/assets/js/bower_components/backbone.wreqr/src/build/backbone.wreqr.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.channel.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.commandStorage.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.commands.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.eventaggregator.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.handlers.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.radio.js
        public/assets/js/bower_components/backbone.wreqr/src/wreqr.requestresponse.js
        public/assets/js/bower_components/backbone/backbone-min.js
        public/assets/js/bower_components/backbone/backbone.js
        public/assets/js/bower_components/backbone/examples/backbone.localStorage.js
        public/assets/js/bower_components/backbone/examples/todos/todos.css
        public/assets/js/bower_compo
Aborting
'''

**Comment by hallshouse** [comment]
FYI: On the weekend, I had a lengthy error message leading to an abort when installing my Dev server from scratch. The error (and abort) occurred each time at this same step for the 3 installs I tried (rebuilt Ubuntu and all). During these installs, I did not install Composer or Bower before LL. During my last install, I followed my standard steps and (installing Composer and Bower in the process) and every thing went smooth (no errors at all). This was for v1.5.2. I've since (today) done a git merge without issue.

**Comment by ht2** [comment]
Hi @davidpesce,

We also hit this issue - it is a direct result of the bower components no longer being ignored, rather they are now included to reduce installation steps (as of #668). 

The most simple solution is to remove the bower_components directory with one simple command prior to merging...

```
rm public/assets/js/bower_components/ -rf
```

That will clear out the existing components that bower will have installed, they will be replaced once you run the merge.

**Comment by ryansmith94** [comment]
Going to add this to the release notes :+1: thanks for reporting your issues @davidpesce and @hallshouse, really appreciated.
